bundles/org.openhab.ui.basic/src/main/resources/OH-INF/i18n/basic.properties

```diff
@@ -1,41 +1,47 @@
+ui.config.basic.biggerFontSize.label = Bigger Font Size
+ui.config.basic.biggerFontSize.description = Displays texts in UI with a bigger font size than the default.
 ui.config.basic.capitalizeValues.label = Capitalize Values
 ui.config.basic.capitalizeValues.description = Displays all widget states in uppercase.
-ui.config.basic.capitalizeValues.option.true = Enable
-ui.config.basic.capitalizeValues.option.false = Disable
+ui.config.basic.chartDPI.label = Chart DPI
+ui.config.basic.chartDPI.description = Adjusts the size of the content (text, line) in the chart.
+ui.config.basic.chartDPI.option.96 = 96 DPI (default)
+ui.config.basic.chartDPI.option.120 = 120 DPI
+ui.config.basic.chartDPI.option.144 = 144 DPI
+ui.config.basic.chartDPI.option.168 = 168 DPI
+ui.config.basic.chartDPI.option.192 = 192 DPI
+ui.config.basic.chartDPI.option.240 = 240 DPI
+ui.config.basic.chartSize.label = Chart Size
+ui.config.basic.chartSize.description = Sets the size of the chart.
+ui.config.basic.chartSize.option.default = Default
+ui.config.basic.chartSize.option.360x180 = 360 x 180 (phone)
+ui.config.basic.chartSize.option.480x240 = 480 x 240 (tablet)
+ui.config.basic.chartSize.option.600x300 = 600 x 300 (tablet)
+ui.config.basic.chartSize.option.720x360 = 720 x 360 (desktop)
+ui.config.basic.chartSize.option.840x420 = 840 x 420 (desktop)
+ui.config.basic.chartSize.option.960x480 = 960 x 480 (desktop)
 ui.config.basic.condensedLayout.label = Condensed Layout
 ui.config.basic.condensedLayout.description = When enabled, changes the layout so that more widgets can fit on the screen.
-ui.config.basic.condensedLayout.option.true = Enable
-ui.config.basic.condensedLayout.option.false = Disable
 ui.config.basic.defaultSitemap.label = Default Sitemap
 ui.config.basic.defaultSitemap.description = The sitemap to show if no parameter is passed.
 ui.config.basic.enableIconify.label = Enable Iconify Icons
 ui.config.basic.enableIconify.description = If enabled, the UI will render iconify icons directly by downloading them from the Internet and caching them in the WEB browser. If disabled, the UI will use the standard method by requesting the server icon proxy and assuming iconify icons are installed on the server. Default is disabled.
```

Comment:
@lolodomo You wrote in the PR description: "The iconify setting is now enabled by default". So I guess you need to update this string?

Reply:
Fixed in PR #2660